oak-parent/pom.xml

```diff
@@ -218,6 +218,7 @@
                   <filter>!org.apache.jackrabbit.oak.plugins.observation.filter</filter>
                   <filter>!org.apache.jackrabbit.oak.plugins.tree.factories</filter>
                   <filter>!org.apache.jackrabbit.oak.plugins.value</filter>
+                  <filter>!org.apache.jackrabbit.oak.plugins.value.jcr</filter>
```

Comment:
Can you please create a separate issue for this change?

Reply:
Sure; see https://github.com/apache/jackrabbit-oak/pull/89.

Reply:
#89 has been merged into trunk.